configure

```diff
@@ -773,7 +773,13 @@ def glob_to_var(dir_base, dir_sub):
     (path, dirs, files) = ent
     for file in files:
       if file.endswith('.cpp') or file.endswith('.c') or file.endswith('.h'):
-        list.append('%s/%s' % (dir_sub, file))
+        srcfile = '%s/%s' % (dir_sub, file)
+        if patch_dir:
```

Comment:
@thefourtheye when this was first landed, I thought I had recommendation to just use `/` as it 'always works'. The output of this function ends up as input to `gyp`, so keeping forward slashes tends to consistency with parallel paths.